common-libs/internal-api-clients/src/helpers/didAuthClientWrapper.ts

```diff
@@ -1,10 +1,15 @@
 import { ThisData as BasicThisData, ClientOptions, createThisData } from './client'
-import { DidAuthAdapter } from './DidAuthAdapter'
 import { DidAuthSession } from './DidAuthManager'
 import { mapFunctions } from './mapFunctions'
 
+export type DidAuthAdapterType = {
+  readonly did: string
+  createDidAuthResponseToken(didAuthRequestToken: string): Promise<string>
```

Comment:
if we change this method to return `Promise<T>` where `T` is an interface that has `isExpiredAt(now: number)` and `toString(): string` methods. It could simplify the code in `common-libs/internal-api-clients/src/helpers/DidAuthManager.ts`